google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -55,8 +55,8 @@ Code inspections for App Engine Java code.</description>
 
     <implicitUsageProvider implementation="com.google.cloud.tools.intellij.appengine.validation.EndpointImplicitUsageProvider"/>
 
-    <checkoutProvider implementation="com.google.cloud.tools.intellij.git.GcpCheckoutProvider"/>
-    <vcsPopupProvider implementation="com.google.cloud.tools.intellij.git.GcpQuickListContentProvider"/>
+    <checkoutProvider implementation="com.google.cloud.tools.intellij.vcs.GcpCheckoutProvider"/>
```

Comment:
Looking at GcpCheckoutProvider, I see a dependency on the `git4idea` package (e.g. `git4idea.GitVcs`). Is this name changed justified still?